fe/fe-core/src/main/java/com/starrocks/statistic/StatisticSQLBuilder.java

```diff
@@ -0,0 +1,111 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.statistic;
+
+import com.clearspring.analytics.util.Lists;
+import com.google.common.base.Joiner;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+
+import java.io.StringWriter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.starrocks.statistic.Constants.STATISTIC_DATA_VERSION;
+import static com.starrocks.statistic.Constants.STATISTIC_HISTOGRAM_VERSION;
+
+public class StatisticSQLBuilder {
+    private static final String QUERY_SAMPLE_STATISTIC_TEMPLATE =
+            "SELECT cast(" + STATISTIC_DATA_VERSION + " as INT), update_time, db_id, table_id, column_name,"
+                    + " row_count, data_size, distinct_count, null_count, max, min"
+                    + " FROM " + Constants.SampleStatisticsTableName
```

Comment:
SampleStatisticsTableName looks wierd, table_statistic_v1 can also support full table analyze

Reply:
Right now, it does look a little weird. However, in the near future PR, the existing code will be deleted. Full data will no longer be stored in table_statistic_v1